src/dos/cdrom.h

```diff
@@ -115,10 +118,67 @@ class CDROM_Interface
 	virtual void InitNewMedia       (void) {}
 };
 
+class CDROM_Interface_SDL final : public CDROM_Interface
+{
+public:
+	CDROM_Interface_SDL(void);
+	CDROM_Interface_SDL(const CDROM_Interface_SDL&);
+	CDROM_Interface_SDL& operator = (const CDROM_Interface_SDL&);
+	virtual ~CDROM_Interface_SDL(void);
+	virtual bool SetDevice(char *path, int forceCD);
+	virtual bool GetUPC(unsigned char &attr, char *upc)
+	{
+		attr = 0;
```

Comment:
`'\0'` literal please.

Reply:
Another big help! I didn't realize the variable is a character.. I thought it held numbers.

Reply:
Actually, I might be wrong, this might be a number. Just had a quick look and saw `Bit8u& attr` in the `CMscdex` class.

Reply:
For this one I think ‘\0’ should be okay. Done.